sips/sip-122.md

```diff
@@ -132,7 +142,7 @@ In the event a synth has triggered a circuit breaker which pauses exchanging, th
 
 - Remove `collateralLiquidated = liquidateInternal(borrower, id, amount)` in the liquidate call in `CollateralShort.sol` and replace with `liquidateFromCollateral()`.
 
-- `collateral.liquidateInternal()` skips the 1 hour interaction delay mechanism.
```

Comment:
Should keep `collateral.liquidateInternal() skips the 1 hour interaction delay mechanism` because of ETH / renBTC backed loans borrowers that have taken out sUSD loan should still be able to fix their loan if ETH / renBTC prices fall.

Reply:
sure i'll update this 👍🏼

Reply:
it's done